.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,23 @@
+---
+name: Feature request
+about: Request a new feature
+labels: enhancement
+---
+
+<!-- If you're not sure on the specifics of the feature or would like a broader
+discussion, please ask consider posting a proposal to
```

Comment:
Should "ask" be removed here?

Reply:
Thank you for catching that 04f46cc323e11a74271ece8dac1fc493280e4595